### DISCUSSION TITLE
Report raw, cumulative rewards in all five lives during training regardless of settings in reward clipping and episodic life in Atari games

### DISCUSSION THREAD

**Discussion by CloudyDory** [thu-ml/tianshou#847] [Q&A]
Hi, it seems that Tianshou currently logs the reward coming after all environment wrappers of Atari games. Is it possible to add an option to log the raw, cumulative rewards of all five lives regardless of settings in reward clipping and episodic life?

**Comment by Trinkle23897** [comment]
https://tianshou.readthedocs.io/en/master/tutorials/cheatsheet.html#handle-batched-data-stream-in-collector
something like this, add an extra wrapper layer for logging right above atari env:
https://github.com/thu-ml/tianshou/blob/4ac407c78f58102fa7f38ded6bfc1e42c703a4a7/docs/tutorials/cheatsheet.rst?plain=1#L223-L253
